whylog/log_reader/exceptions.py

```diff
@@ -7,6 +7,7 @@ class LogReaderError(WhylogError):
 
 class NoLogTypeError(LogReaderError):
     def __init__(self, front_input):
+        super(NoLogTypeError, self).__init__()
```

Comment:
The same question these exceptions. Why they use super if constructor of inherited classes do nothing?